x-pack/plugins/security_solution/cypress/e2e/detection_alerts/ransomware_alerts_display.cy.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setStartDate } from '../../tasks/date_picker';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { login, visit, visitWithoutDateRange } from '../../tasks/login';
+
+import { ALERTS_URL, TIMELINES_URL } from '../../urls/navigation';
+import { ALERT_DATA_GRID, SELECT_HISTOGRAM, TREND_CHART_LEGEND } from '../../screens/alerts';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { TIMELINE_QUERY } from '../../screens/timeline';
+
+describe('Ransomware alerts display', () => {
+  before(() => {
+    login();
+    esArchiverLoad('ransomware');
```

Comment:
The archive used here is really big. It generates 24 alerts when we just need a couple of them to perform the tests. We need to take into consideration that the bigger the archive is, the longer it takes for the method to be executed making the overall test execution slower.

For this specific case, I would generate 2 different archives, one for detection and another one for prevention with just one alert each. I would also split the tests into 2 different contexts, prevention and detection.